internal/pkg/service/stream/sink/type/tablesink/keboola/bridge/file.go

```diff
@@ -203,15 +204,18 @@ func (b *Bridge) importFile(ctx context.Context, file plugin.File, stats statist
 			return err
 		}
 
-		// Save job ID to etcd
-		keboolaFile.StorageJobID = &job.ID
```

Comment:
If I understand correctly you're tracking jobs as separate etcd records now. Do we still need this field on File then? If yes, where is it assigned now?

Reply:
I think this should be still there.
Basically, if we import file and check it again, we don't wan to trigger the StorageJob again. I'll add it back.